src/mgmt_conn.c

```diff
@@ -1588,6 +1589,15 @@ process_message(uzfs_mgmt_conn_t *conn)
 			rc = reply_nodata(conn, ZVOL_OP_STATUS_FAILED, hdrp);
 			break;
 		}
+		vol_size = ZVOL_VOLUME_SIZE(zinfo->main_zv);
+		if (resize_data->size < vol_size) {
+			uzfs_zinfo_drop_refcnt(zinfo);
```

Comment:
sorry for bad review last time..
as we are printing zinfo->name, it need to be done with refcount.. cc: @mittachaitu